server/controllers/UserController.js

```diff
@@ -1,27 +1,84 @@
-import pool from '../db.js';
-import bcrypt from 'bcrypt';
-
+import pool from "../db.js";
+import bcrypt from "bcrypt";
+import { jwtTokens } from "../utils/jwt-helpers.js";
 
 export default {
-    async allUsers(req,res) {
-        try {
-           const users = await pool.query('SELECT * FROM users');
-           res.json({users: users.rows})
+  async allUsers(req, res) {
+    try {
+      const users = await pool.query("SELECT * FROM users");
+      res.status(200).json({ users: users.rows });
```

Comment:
Does this not return with the passwords of all the users in the database? Will you like it if facebook had an endpoint that returns your password?

Reply:
Hahahaha, no oo